README.md

```diff
@@ -103,6 +106,9 @@ The supported options are:
 - `cacheDir`: Path to a GitHub API response cache to avoid throttling
   (e.g. `.changelog`)
 
+- `gitHostingServerURL`: Your Github Enterprise Hosting Server URL (Only for Github Enterprise user. If you're using Github, Don't need it.)
+  (e.g. `https://github.mycompany.com`)
```

Comment:
I named it `gitHostingServerURL`. If we support other hosting services such as bitbucket and gitlab, I think we can create additional options such as 'gitHostingServiceType'.